packages/components/AddItem/AddItem.js

```diff
@@ -69,23 +69,10 @@ export const AddItem = ({ className }) => {
   const isOnline = useNavigatorOnline();
 
   // Todo: The Can should get an entity itself
-  const itemSubject =
-    currentTeam?.id === TEAM_TYPE.PERSONAL
-      ? {
-          __typename: PERMISSION_ENTITY.ITEM,
-          // eslint-disable-next-line camelcase
-          create_item: workInProgressList?._permissions?.create_item || false,
-        }
-      : {
-          __typename: PERMISSION_ENTITY.TEAM_ITEM,
-          // eslint-disable-next-line camelcase
-          team_create_item:
-            // eslint-disable-next-line camelcase
-            workInProgressList?._permissions?.team_create_item || false,
-        };
+  const { _permissions } = workInProgressList;
```

Comment:
also need here a `__typename` and put it in `an` prop on 75th raw.

Reply:
🙅

Reply:
All __typename should be implemented in normalizr.